src/app/cli/src/services/config/models.rs

```diff
@@ -309,6 +315,55 @@ impl Default for JupyterConfig {
 
 ////////////////////////////////////////////////////////////////////////////////////////
 
+#[skip_serializing_none]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "camelCase")]
+#[serde(tag = "kind")]
+pub enum DatabaseConfig {
+    Sqlite(SqliteDatabaseConfig),
+    ClientServer(RemoteDatabaseConfig),
+}
+
+impl DatabaseConfig {
+    pub fn sample() -> Self {
+        Self::ClientServer(RemoteDatabaseConfig {
+            provider: DatabaseProvider::Postgres,
```

Comment:
Having both `kind` and `provider` seems a bit redundant. Would it be better to have `kind: Postgres` and `kind: `MySql` as `DatabaseConfig` enum variants? They all can still reuse the same config struct.

Reply:
Done